src/lib/converter/plugins/CategoryPlugin.ts

```diff
@@ -156,14 +157,16 @@ export class CategoryPlugin extends ConverterComponent {
                 defaultCat.children.push(child);
                 return;
             }
-            let category = categories.find((cat) => cat.title === childCat);
-            if (category) {
+            for (const childCat of childCategories) {
+                let category = categories.find((cat) => cat.title === childCat);
+                if (category) {
+                    category.children.push(child);
+                    return;
```

Comment:
I suspect here's your bug - you meant to `continue` instead of `return`

Reply:
Damn! I missed that :/ Fixed it now.